include/vcpkg/base/messages.h

```diff
@@ -1250,12 +1320,20 @@ namespace vcpkg
                     "{value} is a localized message name like LocalizedMessageMustNotEndWithNewline",
                     "The message named {value} ends with a newline which should be added by formatting "
                     "rather than by localization.");
+    DECLARE_MESSAGE(LocalPortfileVersion,
+                    (),
+                    "",
+                    "Using local portfile versions. To update the local portfiles, use `git pull`.");
+    DECLARE_MESSAGE(ManifestFormatCompleted, (), "", "Succeeded in formatting the manifest files.");
+    DECLARE_MESSAGE(ManifestWithNoTopLevelObj,
+                    (msg::path),
+                    "",
+                    "Failed to parse manifest at {path}: manifest files must have a top-level object");
```

Comment:
Should we combine with other "top level object" messages into "Failed to parse {path}, expected a top-level object" ?